ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSmain.cpp

```diff
@@ -245,18 +245,14 @@ int WINAPI WinMain (HINSTANCE hInstance,HINSTANCE hPrevInstance,LPSTR lpCmdLine,
     // We won't limit those functions like GM, just the default.
     if (wid > screen_width) wid = screen_width;
     if (hgt > screen_height) hgt = screen_height;
-    // Create the window initially without the WS_VISIBLE flag until we've loaded all of the resources.
-    // This will be handled by game_start in roomsystem.cpp where window_default(true) sets the initial
-    // fullscreen state of the window before showing it.
-    enigma::hWnd = CreateWindow("EnigmaDevGameMainWindow", "",
-                                (enigma::getwindowstyle() & ~(WS_VISIBLE)),
-                                (screen_width-wid)/2, (screen_height-hgt)/2, wid, hgt,
-                                NULL, NULL, hInstance, NULL);
+      enigma::hWnd = CreateWindow("EnigmaDevGameMainWindow", "", (enigma::getwindowstyle() & ~(WS_VISIBLE)), (screen_width-wid)/2, (screen_height-hgt)/2, wid, hgt, NULL, NULL, hInstance, NULL);
```

Comment:
I hate to nitpick, but this one bugs me because it breaks our style, the line is too long. Also, I really liked the comment I had explaining why WS_VISIBLE is removed from the window style when initially displayed. Otherwise, I have no other complaints about these changes.

Reply:
since josh already merged it, i will need to make a new pr. I hate having to do this because it makes my "contributions" less useful if they have to be reviewed over and over. But i guess that's my fault for not doing it right the first time. I'm not sure why I removed that comment, it must've been by accident.

Reply:
Don't take it too harsh because even with my improved git skills these days I still make mistakes like this, just because it's hard to track every little thing by the sheer size of some of my prs.

In fact, check out this one I made yesterday:
https://github.com/enigma-dev/enigma-dev/pull/1210

Josh ended up rewriting the history so that the `liblodepng.a` never existed in the first place, instead of merging that pr. I think mainly these mistakes come from being a little bit hasty.

Again, don't take it too serious, because everybody's bound to make mistakes, and this isn't a big one, just try not to do that again. Also, you don't need to make a pr right away, you could ask Josh to rewrite the history of your first PR so that you "didn't" technically make that change, or you could just add the comment back the next time you need to change that file. Again, don't worry about it.